validator/beacon/service.go

```diff
@@ -171,7 +170,7 @@ func (s *Service) listenForAssignmentChange(client pb.BeaconServiceClient) {
 
 		if err != nil {
 			log.Errorf("Could not receive latest validator assignment from stream: %v", err)
-			continue
+			break
```

Comment:
This causes the loop to retry immediately and can cause the logs to spam out. If this breaks, we should just break out of the loop.